### PR TITLE
New Node Progress

### PR DESCRIPTION
**Pull request by josephkane**
* show task logs and steps for nodes added to a cluster

* fix for https://github.com/supergiant/control/issues/1103

### CODE REVIEW

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts [hunk 1]
```diff
@@ -269,9 +268,42 @@ export class ClusterComponent implements OnInit, OnDestroy {
...
+    this.activeMachines = new MatTableDataSource(activeMachines);
+    this.activeMachines.sort = this.sort;
+    this.activeMachines.paginator = this.paginator;
+
+    if (nonActiveMachines.length > 0) {
+      let executingTasksObj = {};
+
+      this.getKubeStatus(this.clusterId).subscribe(
+        tasks => {
+          let executing = tasks.filter(t => t.status == "executing");
```
Comment (nikage) [inline cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: this looks like non rxjs way(

File: cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts [hunk 2]
```diff
@@ -464,6 +496,7 @@ export class ClusterComponent implements OnInit, OnDestroy {
   }
 
   expandRow = (_, row) => row.hasOwnProperty('detailRow');
+  // expandRow = (_, row) => { console.log("WHEN: ", row); return row.hasOwnProperty('detailRow') };
```
Comment (nikage) [inline cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: pls remove commented code

Comment (josephkane) [reply cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts]: thanks, good catch

**Comment by eldridgeh** [comment]
@josephkane 
1. Once the node is provisioned the provisioning section does not go away. I had to refresh the page. This happens in DO and AWS.
screen shot 2018-12-05 at 12 25 43 pm
2. It takes a few seconds for the provisioning section to show up when adding a node to a DO. Would it be possible to make it show up faster? This looked to be instant with AWS
3. I created a cluster in AWS and when it switched over to the cluster details page, the provisioning node table showed up showing that the only node in the cluster was provisioning. We should only see the provisioning table when adding a node. It did not do this with DO. 
4. This is more than likely related to 1, after deleting a node the provisioning table did not go away.

**Comment by josephkane** [comment]
@eldridgeh 
1. great catch, fixed
2. This seem to be a symptom of A.) no "planned" for DO nodes or B.) DO node objects aren't created at the same time as AWS nodes are in the DB. This one is sort of out of my hands (I could be mistaken)
3. This is most likely due to the cluster being declared "operational" before the last node is finished provisioning. In that case, this table would work as designed (because that node actually is still provisioning)
4. fixed with 1

**Pull request merged**